race/fbd117; Crablite Race.json

```diff
@@ -0,0 +1,147 @@
+          "cha": 1
+        }
+      ],
+      "size": ["M"],
+      "speed": {
+        "walk": 30,
+        "swim": 30
+      },
+      "languageProficiencies": [
+        {
+          "common": true,
+          "other": true
+        }
+      ],
+      "age": {
+        "mature": 15,
+        "max": 200
+      },
+      "heightAndWeight": {
+        "baseHeight": 60,
+        "heightMod": "1d12",
+        "baseWeight": 200,
+        "weightMod": "2d6"
+      },
+      "entries": [
+        {
+          "name": "Age",
+          "type": "entries",
+          "entries": [
+            " Crablites reach maturity around age 15 and can live up to 200 years."
+          ]
+        },
+        {
+          "name": "Alignment",
+          "type": "entries",
+          "entries": [
+            "Crablites are generally chaotic, due to being raised removed from society. They also tend to be neutral and prefer to be left alone."
+          ]
+        },
+        {
+          "name": "Size",
+          "type": "entries",
+          "entries": [
+            "Crablites are between 5 and 6 feet tall and weigh between 200 and 300 pounds. Your size is Medium."
+          ]
+        },
+        {
+          "name": "Speed",
+          "type": "entries",
+          "entries": [
+            "Your base walking speed is 30 feet. You have a swimming speed of 30 feet."
+          ]
+        },
+        {
+          "name": "Amphibious",
+          "type": "entries",
+          "entries": ["You can breathe air and water."]
+        },
+        {
+          "name": "Natural Armor",
+          "type": "entries",
+          "entries": [
+            "While you wear no armor, your AC is equal to 12 + your Constitution modifier. You may wield a shield and still gain this benefit."
+          ]
+        },
+        {
+          "name": "Crushing Claws",
+          "type": "entries",
+          "entries": [
+            "Your unarmed strikes inflict bludgeoning damage equal to 1d6 + your Strength modifier. You crush your foes with your claw."
+          ]
+        },
+        {
+          "name": "Clumsy Claws",
+          "type": "entries",
+          "entries": [
+            "Doing anything gentle and delicate with your hands is quite challenging unless you have spent considerable time training them to do a specific task (i.e casting spells, using certain tools, writing, etc...)."
+          ]
+        },
+        {
+          "name": "Death Grip",
+          "type": "entries",
+          "entries": [
+            "You have advantage on any Strength checks or saving throws related to gripping something."
+          ]
+        },
+        {
+          "name": "Languages",
+          "type": "entries",
+          "entries": [
+            "You can speak, read, and write Common and one other language that you and your DM agree is appropriate for your character.."
+          ]
+        }
+      ],
+      "traitTags": ["Natural Weapon", "Amphibious", "Natural Race"],
```

Comment:
```suggestion
      "traitTags": ["Natural Weapon", "Amphibious"],
```

Reply:
I meant to put in natural armor, I changed it